stdlib/public/SDK/Foundation/Decimal.swift

```diff
@@ -13,6 +13,8 @@
 @_exported import Foundation // Clang module
 import _SwiftCoreFoundationOverlayShims
 
+// The methods in this extension exist to match the protocol requirements of FloatinPoint, even if we can't conform directly.
+// If it becomes clear that conformance is truly impossible, we can deprecate some of the methods (e.g. isEqual(to:)) in favor of operators.
```

Comment:
Added some comments to highlight why we have these methods. I was looking at `Decimal` to figure out why we need both `isEqual(to:)` and `==` (and `isLess(than:)` and `<`), and the answer is our pseudo-conformance to `FloatingPoint`; both of the methods come from there.